types/react-imgpro/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for react-imgpro 1.3
+// Project: https://github.com/nitin42/react-imgpro
+// Definitions by: Carlos Li <https://github.com/echoulen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+///<reference types="react" />
+import * as React from "react";
+
+import ProcessImage = ReactImgpro.ProcessImage;
+export default ProcessImage;
+
+declare namespace ReactImgpro {
+	type Mode = "neighbor" | "bilinear" | "bicubic" | "hermite" | "bezier";
+	type Shape = {
+        width?: number;
+        height?: number;
+		mode?: Mode;
+	};
+	type Size = {
+        width: number;
+        height: number;
+        mode:
+            "horizontal_left" |
+            "horizontal_center" |
+            "horizontal_right" |
+            "vertical_top" |
+            "vertical_bottom" |
+            "vertical_middle";
+    };
+	interface ProcessImageProps {
+        image: string;
+        resize?: Shape;
+        quality?: number;
+        greyscale?: boolean;
+        normalize?: boolean;
+        invert?: boolean;
+        opaque?: boolean;
+        sepia?: boolean;
+        dither565?: boolean;
+        scale?: boolean;
+        scaleToFitImage?: {
+            width: number;
+            height: number;
+		};
```

Comment:
This document is mis-formatted all over the place - please get the braces to line up, etc.

Reply:
Thank you, I found the reason which should not indent by `tab`